src/Button/Button.tsx

```diff
@@ -170,6 +126,14 @@ export class Button extends React.PureComponent<Button.Props> {
     style: {}
   };
 
+  componentDidMount() {
+    if (process.env.NODE_ENV !== "production") {
+      if (!satisfyAll(...propsInvariants)(this.props)) {
```

Comment:
is this spread useful?

Reply:
not sure, I left it as I found it in the code previously